web-common/src/features/dashboards/pivot/PivotExpandableCell.svelte

```diff
@@ -22,7 +22,14 @@
       </div>
     </button>
   {/if}
-  {value === "LOADING_CELL" ? "" : value}
+
+  {#if value === "LOADING_CELL"}
+    {""}
+  {:else if value === ""}
+    {"<empty string>"}
```

Comment:
is it not possible to display just blank instead of printing this ?

Reply:
Displaying blank string messes up the styling and also is not supported by the table framework we use. `null` is already a reserve keyword so went with `<empty string>` which is adopted by some browser and tools when you print an empty string.

Reply:
@mindspank thoughts?

Reply:
may be I can merge for now and product can decide correct semantics here

Reply:
Switched to NBSP unicode character after discussing with Alex